vllm/inputs/utils.py

```diff
@@ -0,0 +1,8 @@
+'''Utility functions for input types'''
+
+
+def has_required_keys(
```

Comment:
Perhaps it would be cleaner to put this in `vllm/inputs/data.py` for now, since that's the only place the function is being used.

Reply:
Done.